Virtual method not being called in wx.Sizer in 4.1 dev version

Win 10 64, standard CPython 3.7.4

Downloaded and pip installed wxPython-4.1.0a1.dev4340+af214853-cp37-cp37m-win_amd64.whl, found that the method overrides are not being called on wx.Sizer (well, at least RecalcSizes, didn't check others).

```python
import wx

class Sizer(wx.Sizer):
    def CalcMin(self):
        return wx.Size(0,0)
    def RecalcSizes(self):
        print('yeah, got here')

wx.App()
f = wx.Frame()
f.SetSizer(Sizer())
f.Layout()
```

With wxPython 4.0.6, 
```
> python sizer.py
yeah, got here
```

With the dev 4.1,
```
> python sizer.py
Traceback (most recent call last):
  File "sizer.py", line 12, in <module>
    f.Layout()
wx._core.wxAssertionError: C++ assertion ""Assert failure"" failed at ..\..\src\common\sizer.cpp(994) in wxSizer::RecalcSizes(): Must be overridden if RepositionChildren() is not
```

--- a/etg/sizer.py
+++ b/etg/sizer.py
@@ -158,7 +158,7 @@ def run():
         """)
 
     c.addPyMethod('__iter__', '(self)',
-        doc = "A Py convenience method that allows Sizers to act as iterables that will yield their wx.SizerItems.",
+        doc = "A Python convenience method that allows Sizers to act as iterables that will yield their wx.SizerItems.",
         body = "for item in self.GetChildren(): yield item")
 
     c.addPyCode('Sizer.__bool__ = Sizer.__nonzero__') # For Python 3
@@ -176,6 +176,8 @@ def run():
             failure.
             """)],
         )
+    c.insertItemAfter(c.find('RepositionChildren'), m)
+
 
     #---------------------------------------------
     c = module.find('wxBoxSizer')

